rend3-gltf/src/lib.rs

```diff
@@ -271,7 +271,7 @@ pub async fn filesystem_io_func(parent_directory: impl AsRef<Path>, uri: &str) -
         return Ok(data);
     }
 
-    let path_resolved = parent_directory.as_ref().join(&*uri);
+    let path_resolved = parent_directory.as_ref().join(&uri);
```

Comment:
I also took care of a few clippy lints. I probably have a newer clippy version than CI, but I still thing they were good suggestions overall:

- An unnecessary `&*`.
- Several instances of `#[derive(PartialEq)]` where deriving `Eq` was also possible.